packages/http/src/types.ts

```diff
@@ -359,12 +357,21 @@ export interface HttpService {
 
 export interface HttpOperation {
   /**
-   * Route path
+   * The fully resolved uri template as defined by http://tools.ietf.org/html/rfc6570.
+   * @example "/foo/{bar}/baz{?qux}"
+   * @example "/foo/{+path}"
+   */
+  readonly uriTemplate: string;
+
+  /**
+   * Route path.
+   * Not recommended use {@link uriTemplate} instead. This will not work for complex cases like not-escaping reserved chars.
    */
   path: string;
 
   /**
    * Path segments
+   * @deprecated use {@link uriTemplate} instead
```

Comment:
I think we should have a separate change entry for this, as it is a deprecation that impacts emitters

Reply:
I can add another entry, though this exacgt field was kinda only useful internally. Its not really the path segments but just the different pieces used to build the overall path

Reply:
added a new entry